projects/angular-showcase/src/app/business/business-examples/textexpand-showcase/textexpand-showcase.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-12">
+      <h4 class="sbbsc-block">Basic Example</h4>
+      <sbb-textexpand>
```

Comment:
You can remove everyhing around the <sbb-textexpand> tag to achieve a minimal example. If you like, improve it also in public. Thanks :)